### PR TITLE
Fixes skull cookies being invisible

### LINKED ISSUE #88594: skull cookies are invisible
### Client Version:

_No response_

### Issue Summary:

obtain or craft a skull cookie
they are invisible

### Round ID:

[246019](https://scrubby.melonmesa.com/round/246019)

### Test Merge Information:

_No response_

### Reproduction Steps:

see above

### PR DESCRIPTION
**Pull request by SyncIt21**
## About The Pull Request
- Fixes #88594

same fix as #80179

## Changelog
:cl:
fix: skull cookies are visible again
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
Why don't we add a variable like `no_seasonal_changes` to skull and coffin cookies instead?

**Comment by SyncIt21** [comment]
> Why don't we add a variable like `no_seasonal_changes` to skull and coffin cookies instead?

Good question.

It's because skull and coffin cookies are the only 2 subtypes of sugar cookies so applying `no_seasonal_changes` var to both of them is equivalent of the var not doing anything special I.e there are no other subtypes who have this var set to TRUE.

So might as well Remove the var and reduce code overall

**Comment by SyncIt21** [comment]
> Why don't we add a variable like `no_seasonal_changes` to skull and coffin cookies instead?

You maybe right. I assumed sugar cookies are a abstract type that don't undergo seasonal changes. Let me check

File: code/modules/events/holiday/halloween.dm [hunk 1]
```diff
@@ -33,13 +33,21 @@
 	icon_state = "skeletoncookie"
 	crafting_complexity = FOOD_COMPLEXITY_2
 
+/obj/item/food/cookie/sugar/spookyskull/Initialize(mapload, seasonal_changes = TRUE)
```
Comment (Ghommie) [inline code/modules/events/holiday/halloween.dm:36]:
```suggestion
/obj/item/food/cookie/sugar/spookyskull/Initialize(mapload, seasonal_changes = FALSE)
```

**Pull request merged**